Source/Microsoft.Teams.Apps.CompanyCommunicator/ClientApp/src/actions/index.ts

```diff
@@ -41,4 +43,13 @@ const formatNotificationDate = (notificationDate: string) => {
         notificationDate = notificationDate.replace(',', '\xa0\xa0');
     }
     return notificationDate;
+}
+
+const formatNotificationSendingDuration = (SendingStartedDate: string, SentDate: string) => {
+    let sendingDuration: string = "";
```

Comment:
I don't think you need the ": string" here.

Reply:
done